hazelcast/src/main/java/com/hazelcast/config/NativeMemoryConfig.java

```diff
@@ -103,11 +106,35 @@ public MemorySize getSize() {
      * When native memory region is completely allocated and in-use, further allocation requests will fail
      * with {@link NativeOutOfMemoryError}.
      *
-     * @param size memory size
+     * @param capacity memory size
+     * @return this {@link NativeMemoryConfig} instance
+     * @deprecated Since 5.2, use {@link #setCapacity(Capacity)} instead.
+     */
+    @Deprecated
+    public NativeMemoryConfig setSize(MemorySize capacity) {
+        this.capacity = isNotNull(capacity, "size");
```

Comment:
If have 2 instances, one sets the `capacity` via `setCapacity`, the other via `setSize`, are they still equal??

Maybe add a test into `NativeMemoryConfigTest`

Reply:
Yes, they are equal. Added test in 316de23b436bfd2578cf7ef48ea185be80dcfe23